app/controllers/competencies_controller.rb

```diff
@@ -12,14 +13,7 @@ def index
   # GET /competencies/1
   # GET /competencies/1.json
   def show
-    respond_with @competency do |format|
-      format.json do
-        render json: {}, status: :ok
-      end
-      format.html do
-        redirect_to @competency
-      end
-    end
+    render json: Competency.find(params[:id])
```

Comment:
In fact, why do we have to write anything at all? The default scaffold usually leaves this method empty, because @competency is initialized in set_competency anyway.